jenkins.check_copyright.py

```diff
@@ -30,13 +31,31 @@
     print("File does not exist:", file_name, "(not necessarily an error)")
     exit(0)
 
+def report_incorrect(file_name, pairs):
+    # found a problem so report the problem to the caller and exit
+    print(file_name, "... does not contain a correct Microsoft copyright notice.")
+    print()
+    # print the relevant lines to help the reader find the problem
+    for (regex, line) in pairs:
+        print("    ", line, end="")
+    print()
+    exit(1)
```

Comment:
I would move the call to `exit` outside this function since the purpose here is the report. That way, one won't need to read this function when he's looking at the main algorithm of the script.